src/SFA.DAS.EAS.Portal.Client/Types/Account.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.EAS.Portal.Types
```

Comment:
Now that the types have been moved into the client project, it'd be nice to rename the namespaces to match the new file locations.

Reply:
Done